helm-v3/001-charts.md

```diff
@@ -215,3 +219,43 @@ This can be schematized into something like this:
 In many cases, the base types in a YAML file can be used to derive a base
 schema. Thus it may be desirable to automatically derive schemata if they are
 not supplied.
+
+## Version Reference System
+
+In order to make things look and feel more like Docker, which many people have become comfortable with,
+the `--version` flag will be removed on all subcommands (with the exception of `helm package`).
+
+Referencing a specific chart version can be achieved using a new version reference system.
+Very simply, a chart version can be specified using `NAME[:VERSION]`.
+
+The following commands in Helm 2:
+
+```
+helm fetch stable/hackmd --version 0.1.0
+helm search stable/hackmd --version 0.1.0
+helm inspect stable/hackmd --version 0.1.0
+helm install stable/hackmd --version 0.1.0
+helm upgrade hackmd stable/hackmd --version 0.1.0
+```
+
+Will instead look like the following in Helm 3:
+
+```
+helm pull stable/hackmd:0.1.0
+helm search stable/hackmd:0.1.0
+helm inspect stable/hackmd:0.1.0
+helm install stable/hackmd:0.1.0
```

Comment:
--name hackmd

Reply:
The release name is required in the `install` command of helm 3, see https://github.com/helm/helm/pull/4858. So the `install` should look like this:
``` bash
helm install hackmd stable/hackmd:0.1.0
```